src/java.base/share/classes/java/math/BigDecimal.java

```diff
@@ -79,17 +78,23 @@
  * {@code BigDecimal} arithmetic are broadly compatible with selected
  * modes of operation of the arithmetic defined in ANSI X3.274-1996
  * and ANSI X3.274-1996/AM 1-2000 (section 7.4).  Unlike those
- * standards, {@code BigDecimal} includes many rounding modes, which
- * were mandatory for division in {@code BigDecimal} releases prior
- * to 5.  Any conflicts between these ANSI standards and the
+ * standards, {@code BigDecimal} includes many rounding modes.
+ * Any conflicts between these ANSI standards and the
  * {@code BigDecimal} specification are resolved in favor of
  * {@code BigDecimal}.
  *
  * <p>Since the same numerical value can have different
  * representations (with different scales), the rules of arithmetic
  * and rounding must specify both the numerical result and the scale
  * used in the result's representation.
- *
+
+ * The different representations of the same numerical value are called
+ * members of the same <i>cohort</i>. The {@linkplain
+ * compareTo(BigDecimal) natural order} of {@code BigDecimal}
+ * considers members of the same cohort to be equal to each other. In
+ * contrast, the {@link equals equals} method requires both the
+ * numerical value and representation to be the same for equality to
+ * hold.
```

Comment:
Note, discussing "representation" is ok here since the context is discussing the representation of BigDecimal (in contrast to the text in Comparable).

Reply:
It might be reasonable to add a bit of rationale here and give an example. I think the reason that members of the same cohort might not be considered `equals()` to one another is that they are not substitutable. For example, consider 2.0 and 2.00. They are members of the same cohort, because

    new BigDecimal("2.0").compareTo(new BigDecimal("2.00")) == 0

is true. However,

    new BigDecimal("2.0").equals(new BigDecimal("2.00"))

is false. They aren't considered `equals()` because they aren't substitutable, since using them in an arithmetic expression can give different results. For example:

    new BigDecimal("2.0").divide(new BigDecimal(3), RoundingMode.HALF_UP)
    ==> 0.7

    new BigDecimal("2.00").divide(new BigDecimal(3), RoundingMode.HALF_UP)
    ==> 0.67

I think that's the right rationale, and a reasonable example to illustrate it. Edit to taste. I think it would be good to have material like this, though, because people's immediate reaction to BD being inconsistent with equals is "well that's wrong." Well, no, it's right, and I think this is the reason.

Reply:
Added example using scale and unscaledValue.